.gitignore

```diff
@@ -1,2 +1,4 @@
 !\.gitignore
 build/
+result/
```

Comment:
When are these files created?

Reply:
Oh, right question, these files are created when '--log-path' is set like I have:
--log-path="path_to_repo/tests/result/"
I think it's good path and rather generic. Where are this files created by default?

Reply:
I wouldn't add such paths (result and dut_config.yml) to .gitignore. They can be kept as a change in your local repo. However we can discuss this later.